overlord/snapstate/backend/ns_test.go

```diff
@@ -111,3 +111,64 @@ func (s *nsSuite) TestDiscardNamespaceSilentFailure(c *C) {
 	c.Assert(err, ErrorMatches, `cannot discard preserved namespaces of snap "snap-name": exit status 1`)
 	c.Check(cmd.Calls(), DeepEquals, [][]string{{"snap-discard-ns", "snap-name"}})
 }
+
+func (s *nsSuite) TestUpdateNamespaceMntFilePresent(c *C) {
```

Comment:
Thanks for making the effort to improve the tests and cut the amount of repeated code. I've applied your suggestion to both functions now.